cirq/ops/three_qubit_gates.py

```diff
@@ -26,11 +26,12 @@
     eigen_gate,
     gate_features,
     pauli_gates,
-    op_tree,
     raw_types,
+    op_tree,
```

Comment:
nit: This should be before `pauli_gates`.

Reply:
fixed